src/client/containers/EditBoardPage/EditBoardPage.container.js

```diff
@@ -1,5 +1,135 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { useUser } from '../../firebase/UserContext';
+import './EditBoardPage.styles.css';
+import ArrowButton from '../../components/ArrowButton/ArrowButton.component';
+import PageTitle from '../../components/PageTitle/PageTitle.component';
+import Inputfield from '../../components/InputComponent/InputComponent';
+import Dropzone from '../DropZone/Dropzone.container';
+import GenericButton from '../../components/GenericButton/GenericButton.component';
+import { ApiError } from '../../ErrorBoundary';
 
-export const EditedBoard = () => (
-  <div className="EditedBoard ">The Edited Board!</div>
-);
+const EditedBoard = () => {
+  const [name, setName] = useState('');
+  const [date, setDate] = useState('');
+  const [loading, setLoading] = useState(true);
+
+  // eslint-disable-next-line no-unused-vars
+  const [error, setError] = useState(null);
+  const { boardId } = useParams();
+  const history = useHistory();
+  const { user } = useUser();
+  const userId = user[0].id;
+
+  const boardInfoURL = `/api/users/${userId}/boards/${boardId}`;
+  useEffect(() => {
+    const getBoardInfo = async () => {
+      try {
+        const apiResponse = await fetch(boardInfoURL);
+        const apiData = await apiResponse.json();
```

Comment:
also converting and extracting the json data must come after the ok check
Like this:
```
 if (!apiResponse.ok) {
          throw new ApiError(apiResponse.statusText, apiResponse.status);
        }

 const apiData = await apiResponse.json();
 setName(apiData[0].title);
setDate(apiData[0].deadline.replace(/\..+/, ''));
setLoading(false);
```

Reply:
Always try and provoke an error to see the outcome. Try with your changes, for instance by calling an incorrect url, and then try with the above fix and see how it goes to the 404 page